src/app/reskin/components/stepper/deposit/amount.tsx

```diff
@@ -0,0 +1,22 @@
+import getEmilyLimits from "@/actions/get-emily-limits";
+import { useQuery } from "@tanstack/react-query";
+
+export function AmountDescription({}) {
+  const { data: minimumDeposit } = useQuery({
+    queryKey: ["emilyLimits"],
+    queryFn: async () => {
+      const response = await getEmilyLimits();
+      const isUnlimited = response.perDepositMinimum === 0;
+      // to handle devenv situations
+      return isUnlimited ? 0.0001 : response.perDepositMinimum / 1e8;
+    },
+  });
+  return (
+    minimumDeposit && (
+      <span className="opacity-60">
+        How much BTC are you transferring over to sBTC? Enter an amount that’s
+        above the minimum ({minimumDeposit} BTC)
```

Comment:
Should we round `minimumDeposit`? To avoid displaying strange numbers (eg a lot of decimals, or scientific notation).

Reply:
Good point I think we could fix it later after feedback assuming that the minimum stays at 0.001 will be okay